scripts/set-native-app-manifests.sh

```diff
@@ -94,6 +95,8 @@ echo -e "<?xml version=\"1.0\" encoding=\"UTF-8\"?>
 <!DOCTYPE plist PUBLIC \"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">
 <plist version=\"1.0\">
 <dict>
+	<key>NSCameraUsageDescription</key>
+	<string>This app uses the camera to verify your id.</string>
```

Comment:
To help the user understand that the camera will not be passively just "on":

```suggestion
	<string>This app uses the camera for specific actions you initiate, such as verifying your identity.</string>
```

Reply:
Notably, this text (or the manifest) will probably need to be configurable or something in the future as it will depend on the app as to whether it needs the permission or not.